compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  grapheneos-builder:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: grapheneos-builder
+    privileged: true
+    # These are the bare minimum to build GrapheneOS
+    environment:
+      - DEVICES_TO_BUILD="redfin oriole lynx"
+      # - MANIFESTS_FOR_BUILD="TQ2A.230505.002.2023060700 TQ2A.230505.002.2023060700 TQ2B.230505.005.A1.2023060700"
+      - BUILD_TARGET=stable
+      # - BUILD_NUMBER=2023060700
+      # - BUILD_DATETIME=1686159583
+      # - BUILD_ID=TQ2A.230505.002
+      - USE_PREBUILT_KERNEL=false
+      - USE_PREBUILT_APPS=false
+      - PACKAGE_OS=true
+      # - APPS_TO_BUILD=all
+      # - SKIP_GRAPHENEOS=false
+      # - BUILD_VANADIUM=false
+      - USE_AOSP_TEST_KEYS=false
+      - OFFICIAL_BUILD=true
+      - NPROC_SYNC=8
+      - NPROC_BUILD=8
+    volumes:
+      - ./grapheneos-tree/:/opt/build/grapheneos
+      - ./local_manifests:/.repo/local_manifests:ro
```

Comment:
Is this path definitely correct?

Reply:
https://gerrit.googlesource.com/git-repo/+/master/docs/manifest-format.md#Local-Manifests

Reply:
I mean why is .repo in / ? i

Reply:
There's a bug with these mounts where `repo` can't start with a permission denied error